config/initializers/rack_attack.rb

```diff
@@ -1,14 +1,18 @@
 require "configuration/configuration_service"
 require "configuration/paas_configuration_service"
+require "configuration/env_configuration_service"
+require Rails.root.join("app/helpers/platform_helper")
+
+configuration_service = PlatformHelper.is_paas? ? Configuration::PaasConfigurationService.new : Configuration::EnvConfigurationService.new
 
 if Rails.env.development? || Rails.env.test?
   Rack::Attack.cache.store = ActiveSupport::Cache::MemoryStore.new
   Rack::Attack.enabled = false
 elsif Rails.env.review?
-  redis_url = Configuration::PaasConfigurationService.new.redis_uris.to_a[0][1]
+  redis_url = configuration_service.redis_uris.to_a[0][1]
```

Comment:
I don't have a solution or know exactly why it's this way, but it's worth mentioning that I don't like to see array entries hard coded.

Reply:
Thanks Aaron - agree that there might be a better way to do this, but we don't think it's worth updating as part of this PR.